env.in

```diff
@@ -54,6 +54,14 @@ else
   PATH="@abs_top_srcdir@/bin":$PATH
 fi
 
+# Tests are calibrated to be compiled (they're much slower when
+# interpreted) so make sure they can be auto-compiled upon 'make
+# check'.
+if test "$XDG_CACHE_HOME" = ""; then
+    XDG_CACHE_HOME="${TMPDIR:-/tmp}/guile-fibers-cache"
```

Comment:
This is terrible w.r.t. security, see my comment in the PR for Guix support.

Reply:
Good idea.  Done in v2 (just pushed).